package-lock.json

```diff
@@ -544,7 +544,7 @@
       "dependencies": {
         "@bigcommerce/node-sass": {
           "version": "git+https://git@github.com/bigcommerce-labs/node-sass.git#34dbe47ad10c908af37f77681f33d1759ea28575",
-          "from": "git+https://git@github.com/bigcommerce-labs/node-sass.git#34dbe47ad10c908af37f77681f33d1759ea28575",
+          "from": "@bigcommerce/node-sass@git+https://git@github.com/bigcommerce-labs/node-sass.git#34dbe47ad10c908af37f77681f33d1759ea28575",
```

Comment:
@jairo-bc do I need to revert this as well?

Reply:
yes please

Reply:
done